sunpy/coordinates/tests/test_metaframes.py

```diff
@@ -161,7 +161,7 @@ def test_base_skycoord(rot_hgs):
     s = SkyCoord(1*u.deg, 2*u.deg, 3*u.AU, frame=f.HeliographicStonyhurst, obstime='2001-01-01')
     r = RotatedSunFrame(base=s)
 
-    assert type(r) == type(rot_hgs[1])
+    assert isinstance(r, rot_hgs[1])
```

Comment:
```suggestion
    assert type(r) == type(rot_hgs[1])  # NOQA
```
This intentionally checks that the type is strictly the same.